checkov/terraform/checks/resource/ncp/RouteTableNATGatewayDefault.py

```diff
@@ -0,0 +1,22 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceCheck
+
+
+class RouteTableNATGatewayDefault(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure Routing Table associated with Web tier subnet have the default route (0.0.0.0/0) defined to " \
+               "allow connectivity "
+        id = "CKV_NCP_20"
+        supported_resources = ('ncloud_route',)
+        categories = (CheckCategories.NETWORKING,)
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        if "destination_cidr_block" in conf.keys() and "target_type" in conf.keys():
+            if conf.get("destination_cidr_block") == ["0.0.0.0/0"] \
+                    and conf.get("target_type") == ["NATGW"]:
+                return CheckResult.PASSED
+        return CheckResult.FAILED
```

Comment:
@pj991207 There's 1 case I don't see in your tests which would cause this check to create a lot of noise. What happens if:
1. The `target_type` is not defined? What is the default?
2. What if the cidr block is not 0.0.0.0/0 and the target_type is not NATGW? This case should pass IMO, but it would fail currently

Reply:
answer : 
 1. ncloud_route, the arguments `target_type` and `destination_cidr_block` is required elements, if `target_type` is not defined, the terrform code is wrong
 2. IMO the code is wrong. I fix it now